radis/pgsearch/tests/test_provider_hybrid.py

```
@@ -0,0 +1,197 @@
+        MockClient.return_value.__exit__.return_value = None
+        MockClient.return_value.embed_query.return_value = _unit_vec(0, dim)
+        result = search(_make_search("pneumothorax", group.pk))
+
+    # r2 is in both vector top-K and FTS hits, so its rrf_score should be the largest.
+    top = result.documents[0]
+    assert top.document_id == r2.document_id
+    assert top.cosine_distance is not None
+    assert top.cosine_distance >= 0.0
+    assert top.rrf_score > 0.0
+    # All later documents have a strictly lower or equal rrf_score.
+    for prev, curr in zip(result.documents, result.documents[1:]):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)